tests/SqlFormatterTest.php

```diff
@@ -138,6 +149,27 @@ public static function formatData(): Generator
         return self::fileDataProvider('format.txt');
     }
 
+    /**
+     * Provide long SQL to assert that the whole process (tokenize and format) has linear scalability.
```

Comment:
When/where is that asserted?

Reply:
The tested length will be increased in #119 where the scalability is fixed. Do you want to assert true linear scalability based on time?

(the idea here is simple - O(n^2) is so slow the test will not finish in reasonable time)

Reply:
Ok so right now, it's tested, but not asserted because not fixed yet, is that it? If yes, the comment is misleading, isn't it?

Reply:
I have removed the misleading phpdoc comment - in this PR we add regular but big test and do not mention anything about scalability.